Description: Incorrect path to include directory in pkg-config (libjxr.pc)
 The pkg-config file for libjxr contains incorrect paths to the headers: it
 refers to paths used by the upstream Makefile (sub-directories of
 /usr/include/libjxr) whereas the Debian package actually installs the headers
 directly into a different directory (/usr/include/jxrlib). Therefore,
 packages that depend on jxrlib via the pkg-config file fail to build.
 .
 Original patch did not apply cleanly, rework it (malat)
Author: Yann Leprince <yann.leprince@cea.fr>
Reviewed-By: Mathieu Malaterre <malat@debian.org>
Forwarded: no
Last-Update: 2022-01-28

Index: jxrlib/libjxr.pc.in
===================================================================
--- jxrlib.orig/libjxr.pc.in
+++ jxrlib/libjxr.pc.in
@@ -9,4 +9,4 @@ Description: A library for reading JPEG
 Version: @JXR_VERSION@
 Libs: -L${libdir} -ljpegxr -ljxrglue
 Libs.private: -lm 
-Cflags: -I${includedir}/libjxr/common -I${includedir}/libjxr/image/x86 -I${includedir}/libjxr/image -I${includedir}/libjxr/glue -I${includedir}/libjxr/test -D__ANSI__ -DDISABLE_PERF_MEASUREMENT @JXR_ENDIAN@
+Cflags: -I${includedir}/jxrlib -D__ANSI__ -DDISABLE_PERF_MEASUREMENT @JXR_ENDIAN@
